spec/lib/annotate/annotate_models_spec.rb

```diff
@@ -931,15 +931,15 @@ def mock_column(name, type, options = {})
 
         let(:options) { { additional_file_patterns: additional_file_patterns } }
 
-        it do
-          expect(subject).to eq(additional_file_patterns)
+        it 'returns additional_file_patterns in the argument "options"' do
+          is_expected.to eq(additional_file_patterns)
         end
       end
 
-      context 'without additional_file_patterns' do
+      context 'when additional_file_patterns is not specified is the options' do
```

Comment:
Same as above with typo

Reply:
Same as before, I mean to put is specified in the options...